src/app/home/Header.tsx

```diff
@@ -0,0 +1,30 @@
+"use client";
+import Link from "next/link";
+import React from "react";
+import styled from "styled-components";
+import { HeaderLogo } from "@/components/HeaderLogo";
+
+type Props = {};
+function Header({}: Props) {
+  return (
+    <nav
+      style={{ zIndex: 999 }}
+      className="fixed top-[1.5rem] px-[1rem] flex items-center justify-between space-x-25"
+    >
+      <Link href="/home">
+        <HeaderLogo />
+      </Link>
+      <h1 className="text-white h-[1.9rem] text-header flex items-center">
+        TV shows
+      </h1>
+      <h1 className="text-white h-[1.9rem] text-header flex items-center">
+        Movies
+      </h1>
+      <h1 className="text-white text-header h-[1.9rem] flex items-center">
+        My List
+      </h1>
```

Comment:
스타일이 같은 태그 3개이므로 헤더 리스트를 변수로 선언한 뒤 map 함수로 렌더링 해도 좋을 것 같아요!!

```suggestion

const HEADER_TEXT=['TV shows', 'Movies', 'My List']

...

{HEADER_TEXT.map((text)=>(
      <h1 key={text} className="text-white h-[1.9rem] text-header flex items-center">
        {text}
      </h1>
))